src/setup/cortx_deploy

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+DEFAULT_LOG_LEVEL=0
+function log {
+    LOG_LEVEL=${DEBUG:-$DEFAULT_LOG_LEVEL}
+    [[ $level -lt $LOG_LEVEL ]] && return
+    echo "LOG: $*"
+}
+function show_usage {
+    echo -e "usage: $(basename $0) -n NODE-ID [-f SOLUTION-CONFIG] [-d LEVEL][-o CORTX-CONFIG-URL]"
+    echo -e "Where:"
+    echo -e "..."
+    echo -e " NODE-ID         : Unique Node ID"
+    echo -e " SOLUTION-CONFIG : Solution-Config directory containing - config.yaml, cluster.yaml and secrets.yaml"
+    echo -e "                   (Default: /etc/cortx/solution)"
+    echo -e " LEVEL           : Debug Log Level. By default Level is 0"
+    echo -e " CORTX-CONFIG-URL: URL of CORTX Configuration. Default is yaml:///etc/cortx/cluster.conf"
+    exit 1
+}
+
+function validate_config {
+    local config=$1
+    if [ ! -f $config ]; then
+        echo -e "ERROR: Config $config not found"
+    fi
+}
+
+function yaml_config {
+    local config=$1
+    echo "yaml://$config"
+}
+
+# Constants
+DEBUG_LOG=false
+NODE_ID=0
+SETUP_ARGS=""
+SOLUTION_CONFIG="/etc/cortx/solution"
+CORTX_CONFSTORE="/etc/cortx/cluster.conf"
+
+log "Solution config - $(yaml_config $CONFIG_INFO)"
+log "Cluster config - $(yaml_config $CLUSTER_INFO)"
+log "Secrets config - $(yaml_config $SECRETS_INFO)"
+
+while [ $# -gt 0 ];  do
+    case $1 in
+    -d )
+        shift 1
+        level=$1;
```

Comment:
Use consistent variable naming, e.g. LEVEL all caps or all small at all the places.

Reply:
done